packages/optimizely-sdk/lib/core/optimizely_config/index.ts

```diff
@@ -74,9 +74,13 @@ export class OptimizelyConfig {
       return resultMap;
     }, {});
 
+    const variableIdMap = OptimizelyConfig.getVariableIdMap(configObj);
+
     const experimentsMapById = OptimizelyConfig.getExperimentsMapById(configObj, featureIdVariablesMap);
```

Comment:
It looks like `getExperimentsMapById` also calls `OptimizelyConfig.getVariableIdMap(configObj);`. 

https://github.com/optimizely/javascript-sdk/blob/66a8cd0a4843f85a2deeebf6e05ffd9ddc5c8ac1/packages/optimizely-sdk/lib/core/optimizely_config/index.ts#L348-L352

Is it possible to pass the new `variableIdMap` value into the `getExperimentsMapById` call as well?


For example, 

```suggestion
    const experimentsMapById = OptimizelyConfig.getExperimentsMapById(configObj, featureIdVariablesMap, variableIdMap);
```

Reply:
Yeah, good catch. We should do that as well. I'll update the PR with this change.